MXP-86 Addressed box-sizing issue due to padding being taken into con…

jdashley27 opened the pull request:
Needed to address an issue where some code bases were ignoring padding to be considered with the width, whereas the MDL storybook was changing the box size based upon the set padding.

tarikhamilton commented on components/menu/OverflowMenu/style.module.scss:
```diff
@@ -29,13 +29,16 @@
+  left: 60%;
   box-shadow: 0 12px 15px 0 rgba(0, 0, 0, 0.14);
   z-index: 9;
+  -webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
```
You don't need vendor prefixes. All CSS runs through autoprefixer.

jdashley27 replied:
Works for me, thanks

jdashley27 commented:
@simonwebdesign 
I've updated the styles, so it should fall in line with the links you had shared with me.

@tarikhamilton 
Please look over the updated PR, as I removed the vendor prefixes, and added some slight functionality/unit tests to make it fall in line with the links Simon had included.

tarikhamilton commented on components/menu/OverflowMenu/style.module.scss:
```diff
@@ -6,36 +6,47 @@
+}
+
+.overflow-menu__svg--opened {
+  fill: $color-m-primary-blue;
```
Legacy color variable

jdashley27 replied:
Is there a different one I should be using? I do see a different one of $color-blue.

tarikhamilton commented on components/menu/OverflowMenu/style.module.scss:
```diff
@@ -6,36 +6,47 @@
-  width: 40%;
-  height: 60%;
+  width: 36.111111111%;
+  height: 44.444444444%;
```
These look like magic numbers. What are they for?

jdashley27 replied:
They control the ellipsis size. This allows them to scale relatively with the overall overflow button. I could hard code them to be 13px by 16px, but if the circle gets smaller or bigger, it'll lock the ellipsis size.

tarikhamilton commented on components/menu/OverflowMenu/style.module.scss:
```diff
@@ -6,36 +6,47 @@
+  left: 60%;
   box-shadow: 0 12px 15px 0 rgba(0, 0, 0, 0.14);
   z-index: 9;
+  box-sizing: border-box; /* Addresses padding affecting box-width */
```
Remove comment?

jdashley27 replied:
I removed the comment.

tarikhamilton commented on components/menu/OverflowMenu/style.module.scss:
```diff
@@ -6,36 +6,47 @@
 .overflow-menu__button {
-  @include circle(26px);
+  @include circle(36px);
   padding: $size-md;
```
Legacy size value used

jdashley27 replied:
Any idea as to what the size should be? From my inspect on zeroheight, it shows as 36px. Link: https://zeroheight.com/73kwjxh8i/p/52140/i/861670
